.github/workflows/release.yml

```diff
@@ -10,10 +10,15 @@ jobs:
     name: Check release required
     if: github.event.pull_request.merged && endsWith(github.repository, '-private') != true
     runs-on:
-      group: Default
+      group: macos-gh
```

Comment:
I assume `macos-gh` has Xcode 15.3 or 15.4 installed, so we shouldn't encounter problems while building iOS target. Let's see it in action